### PR TITLE
Create Donations

### PR DESCRIPTION
**Pull request by Golden-VS**

### CODE REVIEW

File: A4G/Donations [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  #                  and we can always pull the pool ID from the membership 
+  #                  JSONs history.
+
+  recipient: 
+    name: Save the Children
+    address: 30 Orange Street
+    tel: +44 (0)20 3272 0300
+    contact: info@savethechildren.org
+
+  ada: 2145 # <- These must be numbers. Can be decimals though. See below
```
Comment (ResponsibleStaking) [inline A4G/Donations:19]: I beliefe you miss a comma here

File: A4G/Donations [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  usd: 265.3 # Only usd currency accepted, check on xe.com for exchange rate.
+  timestamp: 1599026880
```
Comment (ResponsibleStaking) [inline A4G/Donations:21]: This timestamp reflects the 2.9.2020, which is 10 months ago. Please re-check

File: A4G/Donations [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  timestamp: 1599026880 
+  # ^ This is a unix timestamp of the moment you donated.
+  #   You can go to https://www.unixtimestamp.com/ to calculate your donation.
+
+  files:
+  # ^ a list of files relating to the donation. Usualy photos or screenshots 
+  #   of the receipts.
+  #   Make sure to add only relative paths that begin from the root of the 
+  #   repo (they should include your Ticker name).
+    - TICKR/2020-09-02-TGB.png
```
Comment (ResponsibleStaking) [inline A4G/Donations:30]: Please create file references inside your Pool Folder A4G/...

File: A4G/Donations [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    - TICKR/2020-09-02-photo-of-delivery.png
```
Comment (ResponsibleStaking) [inline A4G/Donations:31]: same here

File: A4G/Donations [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  intermediate_txs:
+  # ^ This is a list of
+    - provider: Paypal # The canonical name of the provider you used
+      tx_id: 53667941B3363534Y # The transaction ID the provider gave you 
+    - provider: Cardano
+      tx_id: 76e6d3338c2ec040b7c991d00e065d048797e2a28efc02ff27b72e51e8be3254
+    - provider: ZCash
+      tx_id: 1619b17dd4ab2e51b707a25f3befa26db593bb1cb8c4fa0102a7202760b82b06
+
+
```
Comment (ResponsibleStaking) [inline A4G/Donations:53]: Remove or fill out these optional attributes.
Generally all comments above can be removed.

**Review by ResponsibleStaking — CHANGES_REQUESTED** [review]

**Review by kiriakos — APPROVED** [review]
LGTM

**Comment by kiriakos** [comment]
Thanks @ResponsibleStaking for pointing the PR in the right direction. All requested changes have been done.

@Golden-VS since you are donating via adapay it would be awesome if you can also include the ada transaction in intermediate txs next time. This way people can see your donation on chain. While it is not mandated by our process, it would go a long way in regards to trustworthiness if we who do crypto based donations do that.

**Review by ResponsibleStaking — APPROVED** [review]
Looks all good. As mentioned adding the TX ID would be good for next time!

**Pull request merged**